test/box/gh-6934-nonprintable-error-message.result

```diff
@@ -3,3 +3,8 @@ box.error.new(box.error.ILLEGAL_PARAMS, 'bad: \x80, ok: 🕷')
 - error: 'console: an exception occurred when formatting the output: expected SCALAR,
     SEQUENCE-START, MAPPING-START, or ALIAS'
 ...
+require('net.box').self:call('ok: 👽, bad: \x8a')
```

Comment:
Why do you need netbox to test this? I see the fix only touches console.lua, not net_box.lua.

Reply:
I want to test both paths in `output_handlers["yaml"]`:
`status == true` is covered with the `box.error.new()` testcase;
and `require('net.box').self:call()` results in `status == false`, also it was mentioned in #6781.

Reply:
But the question was different - why do we need netbox for that? It is not a netbox bug, which means it must be reproducible easier. If that was a console bug, then you can use `console` module. If this was yaml bug, then you can use `yaml` module directly. Otherwise I don't understand why this extra step via netbox is needed?